assets/scripts/util/fetch_nonblocking.js

```diff
@@ -0,0 +1,206 @@
+/* global abortEverything, saveStreetIncomplete */
+import $ from 'jquery'
+
+export let serverContacted
```

Comment:
The same issue with export/import of variable. Actually it will not cause an error here, because only old JS is using it, but we should prepare here for modularization of rest js files.